frontend/src/app/modules/admin/types/type-form-configuration.component.ts

```diff
@@ -30,6 +30,7 @@ export interface TypeGroup {
 }
 
 export const adminTypeFormConfigurationSelector = 'admin-type-form-configuration';
+export const emptyTypeGroup = '__empty';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/119d7b3256857927268f1aa81e94dab2fc6fafd5/frontend/src/app/modules/admin/types/type-form-configuration.component.ts#L34"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>